app/components/program/overview.hbs

```diff
@@ -0,0 +1,60 @@
+<div class="program-overview" ...attributes data-test-program-overview>
+  <h2>
+    {{t "general.overview"}}
+  </h2>
+  <div class="program-overview-content">
+    <div class="block programtitleshort" data-test-short-title>
+      <label for={{concat this.id "short-title"}}>
+        {{t "general.programTitleShort"}}:
+      </label>
+      <span>
+        {{#if @canUpdate}}
+          <EditableField
+            @value={{this.shortTitle}}
+            @save={{perform this.changeShortTitle}}
+            @close={{set this.duration @program.duration}}
```

Comment:
`EditableField` is missing a `@clickPrompt={{t "general.clickToEdit"}}`

![image](https://user-images.githubusercontent.com/1410427/112022785-d88f0300-8aef-11eb-8654-ef1435293a0e.png)

Reply:
✔️